python/tvm/relay/__init__.py

```diff
@@ -43,6 +43,7 @@
 from .op import image
 from .op import annotation
 from .op import vision
+from .op import tensor
```

Comment:
why adding `tensor` into relay namespace?

Reply:
I need to import `clip` into the relay namespace for use in the onnx importer. Decided to follow the same pattern as the vision ops, figured it doesn't hurt to provide access to the rest of the tensor operators from the importers.

Reply:
It's unnecessary to import `tensor` to achieve this. You can use `from ..tensor import XXX` for example in the onnx.py